model_compression_toolkit/exporter/model_exporter/pytorch/fakely_quant_onnx_pytorch_exporter.py

```diff
@@ -70,6 +74,18 @@ def export(self) -> None:
 
         Logger.info(f"Exporting PyTorch fake quant onnx model: {self.save_model_path}")
 
+        # Replace float weight with wrapped quantized weights
+        for layer in self.model.modules():
+            if isinstance(layer, PytorchQuantizationWrapper):
+                weights_quantizers = list(layer.weights_quantizers.values())
+                if len(weights_quantizers) > 0:
+                    quantized_weight = torch.nn.Parameter(layer.get_quantized_weights()[KERNEL]).detach()
```

Comment:
What happens if we have more then Kernel-like biases...etc (hint you have the names inside the quantized wrapper).